src/main/java/frc/robot/Robot.java

```diff
@@ -19,13 +19,6 @@
 public class Robot extends TimedRobot {
 
 
-  // Modes
-  public enum Mode {
-    Sharpshooter,
-    Rebounder,
-    Climber
-  };
-
```

Comment:
Are we ditching the operation modes?

Reply:
I pulled them out because it didn't seem like we would have to properly implement the rebounder mode (for this competition at least).

We could still leave it in if we want to expand on it. Right now all focus should be on the 'sharpshooter' operational style.